format/format_test.go

```diff
@@ -130,6 +130,11 @@ func TestIssue41(t *testing.T) {
 	testChangeCustom(t, file, file, f)
 }
 
+func TestDoSeqLet(t *testing.T) {
+	const file = "doseq_let.txt"
+	testChange(t, file, file)
```

Comment:
If you just want to check that a fixture file is unchanged after formatting, use `testFixture`.

I would say that there's nothing too special about this new test and I'd just add it to the list in `TestFixture`, above.